com.unity.shadergraph/Editor/Generation/Enumerations/IncludeLocation.cs

```diff
@@ -1,6 +1,9 @@
+using System;
+
 namespace UnityEditor.ShaderGraph
 {
     [GenerationAPI]
+    [Serializable]
```

Comment:
Not strictly necessary, enums are serializable by default, but serves as a good reminder that this is a serialized enumeration, if anyone is looking to change it in the future...